plugin.service.gto/addon.xml

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.4" provider-name="Birger Jesch">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.service" library="starter.py" start="startup">
```

Comment:
My suggestion would be to put service at the top as first extension point and use plugin for the second one. Is there a need for this to have a script extension point?